docs/content/Deployment/Platform-Docker.md

```diff
@@ -0,0 +1,349 @@
+
+Create a Docker Compose stack by creating a `docker-compose.yml`. A
+production-ready stack would at minimum consist of:
+
+- One or more Cube.js API instance
+- A Cube.js Refresh Worker
+- A Cube Store Router node
+- One or more Cube Store Worker nodes
+- A Redis instance
+
+An example stack using BigQuery as a data source is provided below:
+
+<!-- prettier-ignore-start -->
+[[info |]]
+| Using macOS or Windows? Use `CUBEJS_DB_HOST=host.docker.internal` instead of
+| `localhost` if your database is on the same machine.
+<!-- prettier-ignore-end -->
+
+```yaml
+version: '2.2'
+
+services:
+  cube_api:
+    image: cubejs/cube:v%CURRENT_VERSION
+    ports:
+      - 4000:4000
+    environment:
+      - CUBEJS_DB_TYPE=bigquery
+      - CUBEJS_DB_BQ_PROJECT_ID=cubejs-bq-cluster
+      - CUBEJS_DB_BQ_CREDENTIALS=<BQ-KEY>
+      - CUBEJS_DB_BQ_EXPORT_BUCKET=cubestore
+      - CUBEJS_CUBESTORE_HOST=cubestore_router
+      - CUBEJS_REDIS_URL=redis://redis:6379
+      - CUBEJS_API_SECRET=secret
+    volumes:
+      - .:/cube/conf
+    depends_on:
+      - cubestore_worker_1
+      - cubestore_worker_2
+      - cube_refresh_worker
+      - redis
+
+  cube_refresh_worker:
+    image: cubejs/cube:v%CURRENT_VERSION
+    environment:
+      - CUBEJS_DB_TYPE=bigquery
+      - CUBEJS_DB_BQ_PROJECT_ID=cubejs-bq-cluster
+      - CUBEJS_DB_BQ_CREDENTIALS=<BQ-KEY>
+      - CUBEJS_DB_BQ_EXPORT_BUCKET=cubestore
+      - CUBEJS_CUBESTORE_HOST=cubestore_router
+      - CUBEJS_REDIS_URL=redis://redis:6379
+      - CUBEJS_API_SECRET=secret
+      - CUBEJS_SCHEDULED_REFRESH_TIMER=true
+    volumes:
+      - .:/cube/conf
+
+  cubestore_router:
+    image: cubejs/cubestore:v%CURRENT_VERSION
+    environment:
+      - CUBESTORE_WORKERS=cubestore_worker_1:10001,cubestore_worker_2:10002
+      - CUBESTORE_REMOTE_DIR=/cube/data
+      - CUBESTORE_META_PORT=9999
+      - CUBESTORE_SERVER_NAME=cubestore_router:9999
+    volumes:
+      - .cubestore:/cube/data
+
+  cubestore_worker_1:
+    image: cubejs/cubestore:v%CURRENT_VERSION
+    environment:
+      - CUBESTORE_WORKERS=cubestore_worker_1:10001,cubestore_worker_2:10002
+      - CUBESTORE_SERVER_NAME=cubestore_worker_1:10001
+      - CUBESTORE_WORKER_PORT=10001
+      - CUBESTORE_REMOTE_DIR=/cube/data
+      - CUBESTORE_META_ADDR=cubestore_router:9999
+    volumes:
+      - .cubestore:/cube/data
+    depends_on:
+      - cubestore_router
+
+  cubestore_worker_2:
+    image: cubejs/cubestore:v%CURRENT_VERSION
+    environment:
+      - CUBESTORE_WORKERS=cubestore_worker_1:10001,cubestore_worker_2:10002
+      - CUBESTORE_SERVER_NAME=cubestore_worker_2:10002
+      - CUBESTORE_WORKER_PORT=10002
+      - CUBESTORE_REMOTE_DIR=/cube/data
+      - CUBESTORE_META_ADDR=cubestore_router:9999
+    volumes:
+      - .cubestore:/cube/data
+    depends_on:
+      - cubestore_router
+
+  redis:
+    image: bitnami/redis:latest
+    environment:
+      - ALLOW_EMPTY_PASSWORD=yes
+    logging:
+      driver: none
+```
+
+## Security
```

Comment:
Maybe we need to point that Cube.js doesn't support HTTP2/3/Quick and it's better to use cube.js behind reverse proxy

Reply:
Yeah good point, maybe I'll add a header specifically for reverse proxying